src/bluesearch/database/article.py

```diff
@@ -75,6 +77,51 @@ def paragraphs(self) -> Iterable[tuple[str, str]]:
             is the section title, the second the paragraph content.
         """
 
+    @property
+    def pubmed_id(self) -> str | None:
```

Comment:
I guess this might be just a matter of convention, but should we maybe stick to a single convention?

* https://stackoverflow.com/a/51710151 
* https://www.python.org/dev/peps/pep-0604/

 All the below should be equivalent:

* `Optional[str]`
* `str | None`
* `Union[str, None]`

I guess `Optional[str]` is something we have used so far, however, as per PER 604`str | None` might be the recommended way in new Pythons.

Anyway, I am not requiring any actions here other than just getting everybody's opinion:) @EmilieDel @pafonta

Reply:
I agree about sticking to a single convention. I switched to `Optional[str]` convention as I think it is the most used one for now. If we consider changing this for `str | None` maybe we can create an issue and change it for the entire package ?